merge_requests.go

```diff
@@ -124,6 +124,26 @@ func (m MergeRequestApprovals) String() string {
 	return Stringify(m)
 }
 
+// MergeRequestDiffVersion represents Gitlab merge request version.
+//
+// Gitlab API docs:
+// https://docs.gitlab.com/ce/api/merge_requests.html#get-a-single-mr-diff-version
+type MergeRequestDiffVersion struct {
+	Id             int        `json:"id"`
+	HeadCommitSha  string     `json:"head_commit_sha,omitempty"`
+	BaseCommitSha  string     `json:"base_commit_sha,omitempty"`
+	StartCommitSha string     `json:"start_commit_sha,omitempty"`
+	CreatedAt      *time.Time `json:"created_at,omitempty"`
+	MergeRequestId int        `json:"merge_request_id,omitempty"`
+	State          string     `json:"state,omitempty"`
+	RealSize       string     `json:"real_size,omitempty"`
+	Commits        []*Commit  `json:"commits,omitempty"`
```

Comment:
I think I miss the `Diffs` field here...

Reply:
thx - I totally missed that one... 't was too late last night...